### PR TITLE
Update the title and body when PR exists

### PR DESCRIPTION
**Pull request by gkuga**
This PR addresses #309 and has breaking change.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -39,6 +39,9 @@ inputs:
   auto-merge:
     description: "Enable auto merge for pull request. Requires auto merging to be enabled in repository settings"
     required: false
+  disable-auto-update-pr:
+    description: "Disable the title and body of existing pull request automatically updated."
+    required: false
```
Comment (gr2m) [inline action.yml:44]: I think I'd prefer the option to be called `update-pull-request-title-and-body`

```suggestion
  update-pull-request-title-and-body:
    description: "Set to true if you want the title and body to be updated for an existing pull request"
    default: false
    required: false
```

**Review by gr2m — COMMENTED** [review]
can you please update [`.github/workflows/test.yml`](https://github.com/gr2m/create-or-update-pull-request-action/blob/65a6b01b19b69d7865fcf484340b06548fc02e64/.github/workflows/test.yml) to test the new functionality?

File: action.yml (hunk 1 above)
Comment (gkuga) [reply action.yml:44]: Thanks for the suggestion. I'll write the test this weekend.

**Comment by gkuga** [comment]
@gr2m Could you check the test for me? I think the following part will be `!(steps.run.outputs.pull-request-number > 0)`.

https://github.com/gr2m/create-or-update-pull-request-action/blob/d344e35e0ac3efc75d7189bfe63d6886813f47de/.github/workflows/test.yml#L31

https://github.com/gr2m/create-or-update-pull-request-action/blob/d344e35e0ac3efc75d7189bfe63d6886813f47de/.github/workflows/test.yml#L62

**Comment by gr2m** [comment]
> @gr2m Could you check the test for me? I think the following part will be `!(steps.run.outputs.pull-request-number > 0)`.

Agreed! But something odd is going on here. When I change it to `!(steps.run.outputs.pull-request-number > 0)` it is still evaluated as `steps.run.outputs.pull-request-number > 0`, without the negation:

https://github.com/gr2m/create-or-update-pull-request-action/runs/4599942375?check_suite_focus=true#step:9:2

I'll investigate some more and file a ticket with GitHub

**Comment by gr2m** [comment]
okay never mind, I realized that while I worked on a pull request, the actions that were executed were still form the default branch. I think the problem is with the`pull_request_target` trigger. Now with the approve runs flow I can change it back to `pull_request` which should checkout the correct branch, I hope

**Pull request closed without merging**